src/tools/genpolicy/rules.rego

```diff
@@ -54,6 +54,7 @@ default AllowRequestsFailingPolicy := false
 # Constants
 S_NAME_KEY = "io.kubernetes.cri.sandbox-name"
 S_NAMESPACE_KEY = "io.kubernetes.cri.sandbox-namespace"
+BUNDLE_ID = "[a-z0-9]{64}$"
```

Comment:
I think we need to add the $ characters to the appropriate data in the Policy - not to the CopyFileRequest rules. My understanding is that this new BUNDLE_ID doesn't work in CopyFileRequest due to this:

"sfprefix": "^$(cpath)/$(bundle-id)-[a-z0-9]{16}-",

Reply:
Ahh now I see the last 16 characters come from the sprefix in the settings. Thanks! updated